### PR TITLE
Migrate to PostCSS 8

### PR DESCRIPTION
**Pull request by hudochenkov**
Some maintenance tasks together with upgrade to PostCSS 8. [Migration guide](https://evilmartians.com/chronicles/postcss-8-plugin-migration).

Plugin changes are better reviewed with whitespace diff disabled.

### CODE REVIEW

**Comment by TrySound** [comment]
Clould you please update prettier and stuff in separate pr? Hard to review espesially from mobile.

**Comment by hudochenkov** [comment]
Sure thing, sorry. https://github.com/TrySound/postcss-inline-svg/pull/78

I'll rebase this PR when dependencies PR is merged.

**Comment by hudochenkov** [comment]
@TrySound it's ready for a review.

**Review by TrySound — APPROVED** [review]
Everything else lgtm

File: lib/index.js [hunk 1]
```diff
@@ -27,102 +26,112 @@ function stringifyInliner(inliner) {
     inliner.node.value = String(inliner.parsedValue);
   }
 }
+module.exports = (opts = {}) => {
```
[the remark this answers is not in the document]
Comment (hudochenkov) [reply lib/index.js:30]: I'm not sure what do you mean by “module level statements”. I've added empty line before this line.

**Comment by hudochenkov** [comment]
@TrySound Can I release `5.0.0`? Changelog entries:

- Migrated to PostCSS 8.
- Dropped Node.js 8 support.

**Comment by TrySound** [comment]
Sure

**Pull request merged**